x-pack/plugins/reporting/server/export_types/printable_pdf/lib/pdf/get_doc_options.ts

```diff
@@ -18,17 +20,6 @@ export function getDocOptions(tableBorderWidth: number): BufferOptions {
         paddingTop: () => 0,
         paddingBottom: () => 0,
       },
-      simpleBorder: {
-        // format is function (i, node) { ... };
-        hLineWidth: () => tableBorderWidth,
-        vLineWidth: () => tableBorderWidth,
-        hLineColor: () => 'silver',
-        vLineColor: () => 'silver',
-        paddingLeft: () => 0,
-        paddingRight: () => 0,
-        paddingTop: () => 0,
-        paddingBottom: () => 0,
-      },
```

Comment:
Removed this, since this layout isn't used as of https://github.com/elastic/kibana/pull/78036